### ISSUE TITLE
NTR: Saccharopine high in urine (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8542]
New term request Saccharopine high in urine (segal_030515150204): Saccharopinuria  UMLS: 3806359

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000283
- label: Elevated urinary saccharopine level
- parents: [Increased urinary non-proteinogenic amino acid level (HP:0033098)](https://hpo.jax.org/app/browse/term/HP:0033098)
- pmid: PMID:571908
- def: The amount of saccharopine in the urine, when normalized to urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8542
- synonyms:

**Comment by MickeySegal** [comment]
I see there is also https://hpo.jax.org/app/browse/term/HP:0034028, so this may be redundant.
